### PR TITLE
Update: TORO-43297 - Removing Secondary Logo and Updating to Latest Figma Designs

### PR DESCRIPTION
**Pull request by ejaz-tapestry**

### CODE REVIEW

**Review by p-musi — APPROVED** [review]

File: src/brands/stuart-weitzman/default/font/face.json [hunk 1]
```diff
@@ -129,6 +129,29 @@
...
+          },
+          "variation-settings": {
+            "value": "'wght' 300, 'wdth' 94, 'opsz' 12"
+          },
+          "local": {
+            "value": ["Helvetica Now Var"]
+          },
+          "woff2": {
+            "value": "https://assets.stuartweitzman.com/na/fonts/HelveticaNowVariableRoman.woff2"
+          }
```
Comment (p-musi) [inline src/brands/stuart-weitzman/default/font/face.json:154]: With this addition, we have 6 unique fonts in this file.  I believe we want to keep this to 3, so FuturaBT-Book, FuturaBT-Medium, and avenir-next-variable-roman would need to be removed.  @aoverstreet-tapestry, do you agree?

Comment (p-musi) [reply src/brands/stuart-weitzman/default/font/face.json:154]: Screen Shot 2022-08-26 at 9 54 06 PM
From Figma

Comment (aoverstreet-tapestry) [reply src/brands/stuart-weitzman/default/font/face.json:154]: @p-musi @ejaz-tapestry I would double check with Stephen but I am pretty sure we will not need Avenir.

**Review by aoverstreet-tapestry — COMMENTED** [review]
I would double check with Stephen about adding Avenir.

**Review by p-musi — APPROVED** [review]

**Review by aoverstreet-tapestry — APPROVED** [review]

**Pull request merged**